### PR TITLE
add bundler-audit Fixes #18

### LINKED ISSUE #18: Consider adding bundler-audit gem
https://github.com/rubysec/bundler-audit

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by mensfeld** [comment]
Could you please update the changelog with more details info about API changes? Some things were changed and now after update we need to change to config file for it.

**Comment by mensfeld** [comment]
We could also change the empty_method to empty_methods - sounds better

File: lib/polishgeeks/dev-tools/commands/base.rb [hunk 1]
```diff
@@ -57,6 +57,16 @@ def ensure_executable!
             validator_class.new(stored_output).validate!
           end
         end
+
+        # @param [String] path from which we want take files
+        # @return [Array<String>] list of files in path with app prefix path
+        # @note if path is a file return array with file path with app prefix path
+        def files_from_path(path)
```
Comment (mensfeld) [inline lib/polishgeeks/dev-tools/commands/base.rb]: shouldnt this method be private? we dont use it as a part of commands api I guess

Comment (nijikon) [inline lib/polishgeeks/dev-tools/commands/base.rb]: Yes there are. Made it private.

**Comment by mensfeld** [comment]
:+1: seems decent and ready to go

**Pull request merged**